omod/src/main/webapp/resources/js/view/operation.js

```diff
@@ -103,9 +103,19 @@ define(
 
             cancelOperation: function() {
                 // TODO: Ensure that the current user can cancel the operation
+                if ($('#showOperationCancelReasonField').val() === 'false') {
```

Comment:
I assume that you have tested that this check works correctly when showing the cancel reason is not enabled? Normally, I would code this to check for true (or !== 'true') rather than look for an explicit 'false', but I don't care as long as it works in its current form.